package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@bedrock/vue-turnstile",
+  "version": "0.0.1",
+  "type": "module",
+  "description": "Vue.js Cloudflare Turnstile component for Bedrock Web apps",
+  "module": "./lib/index.js",
+  "scripts": {
+    "lint": "eslint ."
```

```suggestion
    "lint": "eslint \"**/*.{js,vue}\""
```
This glob pattern will match any `.js` or `.vue` file in any directory within the project.

---

This fixed the linter, thanks!